doc/howtos/building-images.rst

```diff
@@ -230,17 +241,34 @@ The following instructions assume that swupd is not used.
 
 .. TODO: document how to create custom image recipes based on ostro-image.bbclass.
 
+.. _`ostro-image.bbclass`: https://github.com/ostroproject/meta-ostro/blob/master/meta-ostro/classes/ostro-image.bbclass
 
-Installing Additional Packages without swupd
---------------------------------------------
+Installing Additional Packages (without swupd)
+----------------------------------------------
+
+An image derived from ``ostro-image.bbclass`` without additional
+configuration is minimal and establishes a core OS with components
+that must always be present on a device. All additional components beyond
+this minimal configuration must be added explicitly by setting
+``OSTRO_IMAGE_EXTRA_FEATURES`` and/or ``OSTRO_IMAGE_EXTRA_INSTALL`` adding them
+to the image when building without swupd.  
```

Comment:
or can this get merged as is now (with all the other changes made), and updated after the release?  I'll keep watching for comments and make updates to the doc. @igor-stoppa @gvancuts @pohly